### DISCUSSION TITLE
Should a check be added for `files/dir` already exists.

### DISCUSSION THREAD

**Discussion by GajenderI** [codefori/vscode-ibmi#1769] [General]
When copying a file into IFS, there should be a check for whether the file already exists. If it exists what should be the default action? 
Rename the file with `-copy` appended or prompt the user for action.

[Refrence Code](https://github.com/codefori/vscode-ibmi/blob/3bd6a2172ef4fa344ed48b351e785b3d50826120/src/views/ifsBrowser.ts#L242C145-L242C145)

**Comment by GajenderI** [comment]
@worksofliam thoughts?

**Reply by worksofliam** [reply]
It's a good idea! We have some other items to focus on but you are welcome to open a PR too!

**Reply by GajenderI** [reply]
Cool!! I would love to do that.
